### PR TITLE
GUVNOR-3524: Do not expose unsupported features by UI (e.g. scorecards, decision trees)

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-3524

This PR adjusts the applicable ```NewResourceHandler``` to prevent creation of new assets if permission is denied.

### CODE REVIEW

**Comment by manstis** [comment]
Part of an emsemble:

- https://github.com/AppFormer/uberfire/pull/859
- https://github.com/kiegroup/kie-wb-common/pull/1211
- https://github.com/kiegroup/drools-wb/pull/641
- https://github.com/kiegroup/kie-wb-distributions/pull/631

File: drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-client/src/main/java/org/drools/workbench/screens/guided/scorecard/client/handlers/NewGuidedScoreCardHandler.java [hunk 1]
```diff
@@ -65,19 +83,24 @@ public ResourceTypeDefinition getResourceType() {
...
-        scoreCardService.call( getSuccessCallback( presenter ),
-                               new HasBusyIndicatorDefaultErrorCallback( busyIndicatorView ) ).create( pkg.getPackageMainResourcesPath(),
-                                                                                                       buildFileName( baseFileName,
-                                                                                                                      resourceType ),
-                                                                                                       model,
-                                                                                                       "" );
+    public boolean canCreate() {
+        return authorizationManager.authorize(PERMISSION,
+                                              sessionInfo.getIdentity());
     }
```
Comment (dgutierr) [inline drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-client/src/main/java/org/drools/workbench/screens/guided/scorecard/client/handlers/NewGuidedScoreCardHandler.java]: I recommend using "authorizationManager.authorize(new ResourceRef("GuidedScoreCardEditor",                                               , ActivityResourceType.EDITOR, ResourceAction.READ, sessionInfo.getIdentity())" instead. This way we avoid having to figure out and build  the permission string "editor.read.?" which is a bit low level.

Comment (manstis) [reply drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-client/src/main/java/org/drools/workbench/screens/guided/scorecard/client/handlers/NewGuidedScoreCardHandler.java]: Thanks @dgutierr I've updated the three affected places.

**Comment by manstis** [comment]
@karreiro Could you please review.

**Comment by manstis** [comment]
Jenkins please retest this.

**Pull request merged**